db/migrate/20190819121620_create_restaurants.rb

```diff
@@ -0,0 +1,16 @@
+class CreateRestaurants < ActiveRecord::Migration[5.2]
+  def up
+    create_table :restaurants do |t|
+      t.column "name", :string, limit: 40
```

Comment:
FYI
this could be written as
`t.string :name, limit: 40`
`t.integer :max_time`